clang/lib/AST/ExprConstant.cpp

```diff
@@ -1921,6 +1921,22 @@ static bool EvaluateFixedPointOrInteger(const Expr *E, APFixedPoint &Result,
 static bool EvaluateFixedPoint(const Expr *E, APFixedPoint &Result,
                                EvalInfo &Info);
 
+/// Support for atomic builtins
```

Comment:
```suggestion
/// Support for atomic builtins.
```
Same for all other comments (also, start with a capital letter, etc.).